ccp/src/main/java/com/togitech/ccp/component/TogiCountryCodePicker.kt

```diff
@@ -96,8 +96,21 @@ fun TogiCountryCodePicker(
         mutableStateOf(getDefaultCountryAndPhoneCode(context, fallbackCountry))
     }
     var isNumberValid: Boolean by rememberSaveable { mutableStateOf(false) }
+    var phoneNumberTransformation = remember(langAndCode) {
+        PhoneNumberTransformation(
+            countryDataMap.getOrDefault(langAndCode.first, fallbackCountry).countryCode.uppercase(),
+        )
+    }
 
     OutlinedTextField(
+        value = phoneNumber,
+        onValueChange = {
+            phoneNumber = phoneNumberTransformation.preFilter(it)
```

Comment:
This is the key line for the bug fix!

Changes below in this file are just reordering parameters to match OTF